_themes/rtd-blog/static/local.css

```diff
@@ -86,25 +95,45 @@ div.admonition {
 
 div.admonition p.admonition-title {
     font-weight: bold;
+    margin-top: 0;
 }
 
-table.footnote {
-    margin: 1em;
+dl.footnote {
+  margin: 0.5rem 0;
+}
+
+dl.footnote dt {
+  float: left;
+  margin-left: 1rem;
+  margin-right: 1rem;
+  margin-top: 0;  
+}
+
+dl.footnote dd > p:first-child {
+  margin: 0px;
```

Comment:
On second thought, using floats isn't nice. Should probably be `display: table` or flexbox

Reply:
I think in general we should avoid doing any CSS changes here. It's not work we're going to get value from in the long term.

Reply:
Used `display: table` in order to not wrestle with flexbox :)